server/index.js

```diff
@@ -54,6 +54,114 @@ app.get('/api/products/:productId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/cart', (req, res, next) => {
+  const cartId = req.session.cartId;
+  if (!req.session.cartId) {
+    return [];
+  } else {
+    const sql = `select "c"."cartItemId",
+                "c"."price",
+                "p"."productId",
+                "p"."image",
+                "p"."name",
+                "p"."shortDescription"
+            from "cartItems" as "c"
+            join "products" as "p" using ("productId")
+          where "c"."cartId" = $1
+          `;
+
+    const values = [cartId];
+
+    db.query(sql, values)
+      .then(result => res.status(200).json(result.rows));
```

Comment:
Be sure to add a `.catch()` here in case querying the database fails. We'll want to pass the `err` to `next` so that the error gets logged and the client gets a `500` response.